pubspec.yaml

```diff
@@ -8,13 +8,13 @@ authors:
 description: Bindings of the ReactJS library for building interactive interfaces.
 homepage: https://github.com/cleandart/react-dart
 environment:
-  sdk: '>=1.13.0 <2.0.0'
+  sdk: '>=1.24.3 <3.0.0'
 dependencies:
-  browser: '>=0.9.0 <0.11.0'
-  js: '^0.6.0'
+  js: ^0.6.0
 dev_dependencies:
-  test: '^0.12.18+1'
-
-transformers:
-- test/pub_serve:
-    $include: test/**_test.dart
```

Comment:
I forgot to verify the Dart 1 DDC instructions, we might need to keep the `transformers` section for that to work since it relies on a pub serve instance.

Reply:
Yeah, this actually is needed, but we can even just keep it in and commented out if

Reply:
I vote to leave it out.  No one is running these tests except us... and running DDC using the dart 1 sdk is a tad overkill when we're already running 'em using DDC in Dart 2 in CI, IMO.

Reply:
^ good point, I'm cool with that